src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -143,6 +143,11 @@ function ExamSettings(props) {
     }
   }
 
+  function handleSubmit(event) {
+    event.preventDefault();
```

Comment:
Why do you want to `preventDefault` here?

Reply:
I had to use preventDefault to prevent the page reload. Without it the page still reloads